background.js

```diff
@@ -287,6 +300,34 @@ $(document).ready(function () {
       }
     });
   }
+  
+  /* method to change the anna status icon on page*/
+  function changeStatus(newStatus) {
+    
+    chrome.storage.local.get(/* String or Array */["statusicon"], function(items){
+      
+      console.log("items.statusicon == undefined: " +(items.statusicon == undefined));
```

Comment:
I think  you shouldnt add this in this code, but only for development of the feature

Reply:
uneccessary logs removed